src/examples/tooltip-modal/TooltipModalPlacement.tsx

```diff
@@ -0,0 +1,221 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState, useRef, useCallback, useMemo } from 'react';
+import styled from 'styled-components';
+import { TooltipModal } from '@zendeskgarden/react-modals';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { getColor } from '@zendeskgarden/react-theming';
+import { Button } from '@zendeskgarden/react-buttons';
+
+const StyledWrapperRow = styled(Row)`
+  padding: ${p => p.theme.space.sm};
+  direction: ltr;
+  overflow: auto;
+`;
+
+const StyledWrapperCol = styled(Col)`
+  min-width: 400px;
+`;
+
+const StyledSpacedRow = styled.div`
+  & > * {
+    margin: 0 ${p => p.theme.space.xs};
+  }
+`;
+
+const StyledCenterRow = styled.div`
+  display: flex;
+  justify-content: center;
+  margin: ${p => p.theme.space.xs} 0;
+`;
+
+const StyledCenterColumn = styled.div`
+  min-width: 150px;
+`;
```

Comment:
Please prefer using Garden whenever possible. I took a crack at refactoring this example:

- `TooltipModal` is moved to the top as it is the relevant chunk of code we want to draw attention to
- Applied Garden grid `Row`/`Col` layout
- As a result, RTL layout is more apparent

See https://codesandbox.io/s/cold-wildflower-ucvf9?file=/src/Example.tsx for details. YMMV.

Reply:
Using the Row/Col for a fixed layout didn't fit well with the design example. 

I have updated the example to move the `TooltipModal` implementation to the top as well as included RTL support.

Reply:
Let's double-check this...

@m-lai in order to reduce cognitive distraction and reinforce Garden usage, one of the goals of the website is to use as much Garden as is possible (and tolerable per design). Can you have a look at https://codesandbox.io/s/cold-wildflower-ucvf9?file=/src/Example.tsx and determine if the "pure" Garden grid layout is satisfactory for the example?

If so, the backing codesandbox source would be more desirable than the customizations needed to get the layout 100% locked to the current design. If not, then NBD we can let this one example slide under the goal of "more Garden".

Quick look at the in-page revision...

<img width="746" alt="Screen Shot 2020-09-28 at 2 07 40 PM" src="https://user-images.githubusercontent.com/143773/94486319-1724b000-0194-11eb-89f9-a78b0fd5cedd.png">

Reply:
Please also note that mobile layout is not accommodated as well as it would be by leveraging Garden's grid.

Reply:
@jzempel Hmmm it works for me, I agree we should dogfood our own product using the grid...even though mobile doesn't look as great I'd hope nobody is spending their time trying to grab tooltip modals off mobile lol

Reply:
@jzempel this is now using `react-grid`